### PR TITLE
Feature/update aspnet grpc wcf net 5 

### PR DESCRIPTION
**Pull request by sughosneo**
This PR contains following changes :

- Updates `index` page
- Includes the changes suggested by the @shirhatti in PR #21984

### CODE REVIEW

File: docs/architecture/grpc-for-wcf-developers/docker.md [hunk 1]
```diff
@@ -24,10 +24,9 @@ For each image, there are four variants based on different Linux distributions,
 
 | Image tag(s) | Linux | Notes |
 | --------- | ----- | ----- |
-| 5.0-buster, 5.0 | Debian 10 | The default image if no OS variant is specified. |
-| 5.0-alpine | Alpine 3.9 | Alpine base images are much smaller than Debian or Ubuntu ones. |
-| 5.0-disco | Ubuntu 19.04 | |
-| 5.0-bionic | Ubuntu 18.04 | |
+| 5.0-buster-slim, 5.0 | Debian 10 | The default image if no OS variant is specified. |
+| 5.0-alpine | Alpine 3.12 | Alpine base images are much smaller than Debian or Ubuntu ones. |
+| 5.0-focal| Ubuntu 20.04 | |
```
Comment (IEvangelist) [inline docs/architecture/grpc-for-wcf-developers/docker.md:29]: Consider using the Docs Authoring Pack - Visual Studio Code extension if you're not already. There are plenty of really good features that add support for common MD scenarios, one of them specifically is the formatting or consolidation of tables for readability. For more information, see [Reformat Markdown tables](https://docs.microsoft.com/en-us/contribute/docs-authoring/reformat-table).

Using the **Evenly distribute selected table** open when you select the table:

```md
| Image tag(s)         | Linux        | Notes                                                           |
|----------------------|--------------|-----------------------------------------------------------------|
| 5.0-buster-slim, 5.0 | Debian 10    | The default image if no OS variant is specified.                |
| 5.0-alpine           | Alpine 3.12  | Alpine base images are much smaller than Debian or Ubuntu ones. |
| 5.0-focal            | Ubuntu 20.04 |                                                                 |
```

This formatting makes the table easier to maintain, and read in its MD form.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

File: docs/architecture/grpc-for-wcf-developers/docker.md (hunk 1 above)
Comment (sughosneo) [reply docs/architecture/grpc-for-wcf-developers/docker.md:29]: This extension looks to be cool. No, I wasn't using that but I will use now. Thank you for sharing 👍

**Comment by shirhatti** [comment]
Thanks @sughosneo! Appreciate it!

**Comment by gewarren** [comment]
Friendly reminder to please check for build warnings before merging. Also, consider using the Docs Authoring Pack to insert valid xref links - see the green tip box at https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#xref-cross-reference-links.

**Pull request merged**